tool/npm_template/package.json

```diff
@@ -16,5 +16,12 @@
         "gltf",
         "webgl",
         "3d"
-    ]
-}
\ No newline at end of file
+    ],
+    "devDependencies": {
```

Comment:
Won't these dependencies be blown away by `pub run grinder npm`?

If I'm not mistaken, these will be saved to `build/npm/node_modules/**` and the entire build folder gets blown away by the rebuild process.

Reply:
Could they be installed globally?

Reply:
Assuming we install directories from the same directory where we're going to call `npm run docs`, nothing gets blown away. Currently it's a manual step to call `npm run docs`. So if we only regenerate docs when `index.js` is changed, we can just run `npm install && npm run docs` then, from the template folder, and commit the results. If generating docs happens on each release, then `npm install && npm run docs` will need to be run from that final directory.

Global installs would also work, just less common in the Node.js world. Up to you.

Reply:
In that case, I'd integrate these commands with the main build process. Will update PR in a moment.

Reply:
Nice! Your update looks good to me.

Reply:
Just in case, is language in npm's `README` and `index.js` correct? This may be the first thing newcomers see, so everything must be clear.

Reply:
It looks pretty good to me, I may write up some more details for `index.js` in a future PR.